source/configuration/modules/omfile.rst

```diff
@@ -1,434 +1,441 @@
-   used gzip compression (then, even higher sizes may make sense)
-
-.. function::  dirOwner [userName]
-
-   *Default: system default*
-
-   Set the file owner for directories newly created. Please note that
-   this setting does not affect the owner of directories already
-   existing. The parameter is a user name, for which the userid is
-   obtained by rsyslogd during startup processing. Interim changes to
-   the user mapping are not detected.
-
-.. function::  dirOwnerNum [uid]
-
-   *Default: system default*
-
-   *Available since: 7.5.8, 8.1.4*
-
-   Set the file owner for directories newly created. Please note that
-   this setting does not affect the owner of directories already
-   existing. The parameter is a numerical ID, which is used regardless
-   of whether the user actually exists. This can be useful if the user
-   mapping is not available to rsyslog during startup.
-
-.. function::  dirGroup [groupName]
-
-   *Default: system default*
-
-   Set the group for directories newly created. Please note that this
-   setting does not affect the group of directories already existing.
-   The parameter is a group name, for which the groupid is obtained by
-   rsyslogd on during startup processing. Interim changes to the user
-   mapping are not detected.
-
-.. function::  dirGroupNum [gid]
-
-   *Default: system default*
-
-   Set the group for directories newly created. Please note that this
-   setting does not affect the group of directories already existing.
-   The parameter is a numerical ID, which is used regardless of whether
-   the group actually exists. This can be useful if the group mapping is
-   not available to rsyslog during startup.
-
-.. function::  fileOwner [userName]
-
-   *Default: system default*
-
-   Set the file owner for files newly created. Please note that this
-   setting does not affect the owner of files already existing. The
-   parameter is a user name, for which the userid is obtained by
-   rsyslogd during startup processing. Interim changes to the user
-   mapping are *not* detected.
-
-.. function::  fileOwnerNum [uid]
-
-   *Default: system default*
-
-   *Available since: 7.5.8, 8.1.4*
-
-   Set the file owner for files newly created. Please note that this
-   setting does not affect the owner of files already existing. The
-   parameter is a numerical ID, which which is used regardless of
-   whether the user actually exists. This can be useful if the user
-   mapping is not available to rsyslog during startup.
+People often report problems that dynafiles are not properly created.
+Apart from generating invalid paths/filenames, sometimes these problems are due to SELinux rules, which do not permit
+the creation of those files. The same happens for generic permission issues (this is often a problem
+under Ubuntu where permissions are dropped by default)
+
+=========  ==========================
+template:  "template-name"
+Default:   none
+Legacy:    $ActionFileDefaultTemplate
```

Comment:
This actually is the equivalent of setting it at module load time. The legacy way to specify the template was
*.* /var/log/foo;templatename